commandline.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 Ontario Systems
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package isclib
+
+import (
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Commands represents the ISC command lines that are available
+type Commands uint
+
+func (c *Commands) Set(command Commands)     { *c |= command }
```

Comment:
I would document everything that is public (and therefore part of the API of this library). Does linting pass? I know the linter I currently use would complain about these being undocumented.